backend/src/test/java/gov/cdc/usds/simplereport/db/repository/BaseRepositoryTest.java

```diff
@@ -14,6 +14,14 @@
 @Import(SliceTestConfiguration.class)
 public abstract class BaseRepositoryTest {
 
+    protected static void pause() {
+        try {
+            Thread.sleep(2);
+        } catch (InterruptedException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
```

Comment:
~~Ah, nice. Putting this here means the higher level can't mess it up.~~

~~I suppose a check to verify it's actually **in a transaction** might be too much babysitting that the higher level code hasn't messed up?~~

Reply:
Oh, I misunderstood where this code was... nevermind.